examples/visual-tests/amp-story/amp-story-page-attachment.html

```diff
@@ -49,23 +49,114 @@
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="inline-dark-theme-1-image">
+        <amp-story-page-attachment
+          title="Lorem ipsum"
+          layout="nodisplay"
+          cta-image="noimage"
+          theme="dark"
+          >
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="inline-dark-theme-2-images">
+        <amp-story-page-attachment
+          title="Lorem ipsum"
+          layout="nodisplay"
+          cta-image="noimage"
+          cta-image-2="noimage"
+          theme="dark"
+          >
+          <h1>Some really cool video</h1>
           <p>
-            Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
           </p>
         </amp-story-page-attachment>
       </amp-story-page>
-      <amp-story-page id="page-2">
+
+      <amp-story-page id="outlink-default">
         <amp-story-page-attachment layout="nodisplay" href="https://google.com">
         </amp-story-page-attachment>
       </amp-story-page>
+
+      <amp-story-page id="outlink-custom-text">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          href="https://google.com"
+          cta-text="Custom text">
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="outlink-no-image">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          href="https://google.com"
+          cta-image="none">
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="outlink-dark-theme">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          href="https://google.com"
+          theme="dark">
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="outlink-custom-background">
```

Comment:
I wonder if we should add `-color` to be more descriptive.
At first glance it makes me think if could be a `background-image`.

```suggestion
      <amp-story-page id="outlink-custom-background-color">
```